src/python/pants/engine/internals/synthetic_targets.py

```diff
@@ -22,7 +22,6 @@
     class SyntheticExampleTargetsRequest(SyntheticTargetsRequest):
         path: str = SyntheticTargetsRequest.REQUEST_TARGETS_PER_DIRECTORY
         path: str = SyntheticTargetsRequest.SINGLE_REQUEST_FOR_ALL_TARGETS
-        path: str = "fixed/path/..."  # Get a single request for named directory only.
```

Comment:
I think I prefer to keep this detail as an implementation detail subject to change if need be.